stable/datadog/values.yaml

```
@@ -567,7 +577,10 @@ daemonset:
   ## Allow the DaemonSet to perform a rolling update on helm update
   ## ref: https://kubernetes.io/docs/tasks/manage-daemon/update-daemon-set/
   #
-  # updateStrategy: RollingUpdate
+  # updateStrategy:
+  #   type: RollingUpdate
+  #   rollingUpdate:
+  #     maxUnavailable: 10%
```

A nit here but for consistency with the defaults:

```suggestion
  #     maxUnavailable: "10%"
```